### DISCUSSION TITLE
ThemeManager Material.Avalonia

### DISCUSSION THREAD

**Discussion by CarlosEX** [AvaloniaUI/Avalonia#5568] [Q&A] [answered]
Hey guys, could you give me a light on how I could fix this?

I used ThemeManager and was able to make it work by following the example of the repository:
https://github.com/wieslawsoltes/Avalonia.ThemeManager
![themManager](https://user-images.githubusercontent.com/33513870/109330148-2179d180-783a-11eb-804a-aaa02a091fbf.gif)

But I can't see how I could apply this using the themes of Material.Avalonia. and in this case customizing based on three options, as we can configure at design time:

```
    <Application.Resources>
        <themes:BundledTheme BaseTheme="Light" PrimaryColor="Teal" SecondaryColor="Amber"/>
    </Application.Resources>
```

I'll associate the values with three comboboxes:

1. Combo-1 BaseTheme
1. Combo-1  PrimaryColors
	Red
	Pink
	Purple
	DeepPurple
	...
1. Combo-1  SecondaryColors:
	Red
	Pink
	Purple
	....
	
To individually fill each combobox this would be a principle? I'm lost now!
        public static ThemeSelector? SelectorBaseTheme { get; set; }
        public static ThemeSelector? SelectorPrimaryColors { get; set; }
        public static ThemeSelector? SelectorSecondaryColors { get; set; }

**Comment by CarlosEX** [comment, marked as answer]
Solved! Got.

**Reply by maxkatz6** [reply]
As far as I know, "Avalonia.ThemeManager" project works only with Avalonia Default theme.
How you managed to get it work with Material, or did you wrote your own theme manager implementation?

**Reply by CarlosEX** [reply]
Yes, @maxkatz6'm going to build a color palette and a theme manager. You can access the themes through the classes provided. I'm going to start building today and I believe that in two weeks I'll finish and split it for us.
